tests/failingTests.js

```diff
@@ -12,8 +12,12 @@ window.failingTests = {
     imports: [
       'Javascript'
     ],
-    scope: [
-      'can reference parent items id'
-    ]
+  },
+  Initialize: {
+    Translate: ["can be loaded"],
+    Scale: ["can be loaded"],
+    Rotation: ["can be loaded"],
+    Font: ["can be loaded"],
+    AnimatedImage: ["can be loaded"],
```

Comment:
I don't like how exactly this does look.
If you we use `module.element.options`, could we use `Initialize.module.element` as the test id?
Then, this list will look like this:
```javascript
Initialize: {
  'QtQuick 2.5': [
    'Translate',
    'Scale',
    …
  ]
}